tools/cuds-prioritized-attribution/README.md

```diff
@@ -369,19 +367,12 @@ This means that your project does not contain a VPC. Composer requires a network
 
 This means that your organization has the organization policy of [Domain Restricted Sharing](https://cloud.google.com/resource-manager/docs/organization-policy/restricting-domains) enabled. You must disable the policy constraint before creating an environment. This action permits Cloud Composer to apply the required ALCs to the Cloud Storage bucket for your environment. You can re-enable the policy constraint after environment creation.
 
-3. `Terraform: Cannot remove all owners from a dataset., invalid
-  on main.tf line 71, in resource "google_bigquery_dataset" "corrected_dataset":
-  71: resource "google_bigquery_dataset" "corrected_dataset" {`
-
-  This is expected. This will occur if you enter `terraform apply` more than once. At the time of this release, Terraform does not allow allow additive permissions on BigQuery datasets. This means that it is not possible to update existing ACLs on a per use-case basis. You can ignore this error.
-
-4. `BigQuery fails to execute the query due to different dataset regions`
+3. `BigQuery fails to execute the query due to different dataset regions`
 
 All of the datasets used in this solution must reside in the same region. Your exported billing data most likely resides in your company's location (if you are a European company, it is probably in the EU. If you are in the US, your dataset is probably in the US.) To resolve this, verify that the dataset containing the `commitments_table` and the `cud_corrected_dataset` are in the same region as your billing data. You can update the location of a dataset following these [instructions](https://cloud.google.com/bigquery/docs/locations#moving-data).
 If your billing datasets are all in the same region, ensure that your GCS buckets are also in the same region. You specified this on creation using the `l` flag when executing `gsutil mb`.
 
 
-5. `Other "Invalid Argument"`
+4. `Other "Invalid Argument"`
```

Comment:
nit / FYI just make all your numbered items `1.` markdown is smart enough render them and you save mental cycles updating consecutive numbers.
e.g.
```md
1. one
1. two
1. three
1. four
```
1. one
1. two
1. three
1. four

Reply:
Great idea. I made those changes. 
thanks